src/main/webapp/user-page.html

```diff
@@ -42,11 +42,19 @@
   </head>
   <body onload="initiateUI();">
   <!-- Website title header -->
-    <h1 class="site-heading text-center text-white d-none d-lg-block">
+    <h1 class="site-heading text-center text-white d-none d-lg-block mb-1">
       <span class="site-heading-upper text-primary mb-3">Find Great Local Food</span>
       <span class="site-heading-lower" id="page-title">Website Name</span>
     </h1>
 
+    <!-- About Me -->
+    <div class="text-center" contenteditable="true" id="about-me-container">
```

Comment:
`contenteditable="true"` should probably be on the inner `p` tag, currently the text on the save button can be edited too if you click out of it.

Reply:
oh it is! Thanks a lot for all these reviews.